### PR TITLE
CM-19861: Add SAST scan

### PR DESCRIPTION
**Pull request by sergey-rynkevich**

### CODE REVIEW

File: cli/code_scanner.py [hunk 1]
```diff
@@ -775,6 +775,11 @@ def _map_detections_per_file(detections) -> List[DetectionsPerFile]:
     return [DetectionsPerFile(file_name=file_name, detections=file_detections)
             for file_name, file_detections in detections_per_files.items()]
 
+def _get_file_name_from_detection(detection):
+    if detection['source_policy_type'] == "SAST":
```
Comment (MichalBor) [inline cli/code_scanner.py:779]: this is the old category field, please check `category` value. if you don't get it in the response so add it please

**Review by MichalBor — COMMENTED** [review]
👏🏻

File: cli/code_scanner.py (hunk 1 above)
Comment (sergey-rynkevich) [reply cli/code_scanner.py:779]: it's not `category` - this is `policyType` field

Comment (MichalBor) [reply cli/code_scanner.py:779]: `category` is the name of `policyTypeV2` on the detection entity

**Review by MichalBor — APPROVED** [review]

**Pull request merged**